wiki/Installing-drivers-and-fix-them.md

```diff
@@ -0,0 +1,77 @@
+# Installing and updating drivers 
+In this post, you will learn how to download or update drivers on your computer.
+It's highly recommended to avoid using third-party software to update or install drivers.
+## Before you get into the process, you need to know your computer.
+The first step in installing drivers is to correctly identify the hardware you need drivers for. This information can be found in your computer's manual or by using the Device Manager (Windows) (right click into windows icon and press “Device Manager”). In the Device Manager, look for the device you require drivers for, and expand its entry to see its model number. In System Information, navigate to the “Hardware” section and find the device you require drivers for.
+
+## Method 1: Check Windows Update
+You can usually find drivers and drivers update in Windows Update, instead of manually downloading the file.
+ 1. Open the Settings app using the Start menu.
+ 2. Go to Updates and security.
+ 3. Click on “Check updates”.
+ 4. Windows Update might say that you are up-to-date, even if optional updates are available. In this case, you will see an “Optional Updates” button. Clicking on it will show a list of updates (all of them are typically driver updates). Select all of them and click on “Download and install”.
+
+## Method 2: Download using the manufacturer's website.
+It's highly recommended to download or update the drivers, through the manufacturer's website, as it's guaranteed that they will work correctly. This is especially a good idea for the GPU (Graphics) drivers.
+All you need to do is:
+
+ 1. On the search engine of your choice (Google, Bing, etc…), search for "{device manufacturer} drivers".
+ 2. Next, enter the ID of the device and follow the instructions on the page.
```

Comment:
"Visit the manufacturer's website and enter the model of your device, then follow the instructions on the page."